suppressions.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    # Copyright 2015 WSO2 Inc. (http://wso2.org)
+    #
+    # Licensed under the Apache License, Version 2.0 (the "License");
+    # you may not use this file except in compliance with the License.
+    # You may obtain a copy of the License at
+    #
+    # http://www.apache.org/licenses/LICENSE-2.0
+    #
+    # Unless required by applicable law or agreed to in writing, software
+    # distributed under the License is distributed on an "AS IS" BASIS,
+    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    # See the License for the specific language governing permissions and
+    # limitations under the License.
+-->
+
+<!DOCTYPE suppressions PUBLIC
+        "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+        "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+
+    <suppress checks="Javadoc.*" files=".*/src/test/java/.*" />
+
+    <suppress checks="JavadocPackage" files=".*/src/(main|integration)/java/.*" />
+    <suppress checks="JavadocPackage" files=".*/src/.*/internal/.*" />
+
+    <suppress checks="JavadocStyle" files=".*/src/(main|integration)/java/.*" />
+    <suppress checks="JavadocStyle" files=".*/src/.*/internal/.*" />
+
+    <suppress checks="MethodNameCheck" files=".*/*CommandProvider.java" />
+    <suppress checks="Regexp" files=".*/*CommandProvider.java" />
+
+
+    <!-- copied from apache hadoop, won't fix style to keep diff minimal -->
+    <suppress checks=".*" files=".*/LocalJobRunnerWithFix.java" />
+
+    <!-- do not check thrift generated files -->
+    <suppress checks=".*" files=".*/transaction/distributed/thrift/.*" />
+
+    <!-- do not check swagger generated files -->
+    <suppress checks=".*" files=".*/src/gen/java/.*" />
+
+<!--    do not check auto generated protobuf classes and the service class-->
+    <suppress checks=".*" files=".*/src/test/java/org/wso2/grpc/test/.*" />
+
+
+</suppressions>
```

Comment:
Why, we want to suppress warnings? What are the cases that really need for the grpc? Please clean this file.

Reply:
"org.wso2.grpc.test" package has the protobuf message classes and the Grpc service class that are used in test cases. Those protobuf classes are auto generated by the protoc compiler. That's why I suppressed the warnings of those classes.